### DISCUSSION TITLE
Подраздел 'Продвинутый синтаксис' JS

### DISCUSSION THREAD

**Discussion by vitya-ne** [doka-guide/content#5104] [Разное]
Хочу поднять вопрос о разделе 'Продвинутый синтаксис'.
Возможно, что-то подобное уже обсуждалось или находится в issues. 

Мне кажется, сейчас с ним что-то не так (первое что бросается в глаза):
1. Часть текущих статей точно не относятся к синтаксису ([Поверхностное и глубокое копирование](https://doka.guide/js/shallow-or-deep-clone/), [Date](https://doka.guide/js/date/))
2. Нет отдельной статьи про деструктуризацию (есть частичное описания для массивов в свёрнутом блоке 'На практике' статьи Массивы)

что думаете ?

**Comment by TatianaFokina** [comment]
@HellSquirrel, тут нужно твоё мнение.

**Comment by HellSquirrel** [comment]
Мне кажется что в разделе JavaScript должно быть то как пишется и как выполняется JS. Мне кажется не обязательно ограничивать это только синтаксисом. Даже в спецификации есть штуки не относящиеся в синтаксису. Например https://tc39.es/ecma262/multipage/executable-code-and-execution-contexts.html.
